roles/internal/alpaca/defaults/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+alpaca_from_source: yes
+
+alpaca_version: master
+alpaca_clone_directory: /opt/alpaca
+
+alpaca_karaf_repos: []
+
+alpaca_features:
+  - islandora-http-client
+  - islandora-connector-broadcast
+  - islandora-indexing-triplestore
+  - islandora-indexing-fcrepo
+
+alpaca_karaf_dir: /opt/karaf
+alpaca_karaf_etc_dir: "{{ alpaca_karaf_dir }}/etc"
+alpaca_local_mvn_path: /home/{{ alpaca_user }}/.m2/repository
```

Comment:
It might be a little more Maven-idiomatic to do this with a templated [`settings.xml`](https://maven.apache.org/settings.html) file.